myRobot/src/main/java/MyRobot.java

```diff
@@ -5,28 +5,62 @@
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
+import edu.wpi.first.wpilibj.DistanceEncoder;
 import edu.wpi.first.wpilibj.IterativeRobot;
+import edu.wpi.first.wpilibj.PIDController;
+import edu.wpi.first.wpilibj.Spark;
+import edu.wpi.first.wpilibj.SpeedController;
+import si.uom.SI;
+import systems.uom.common.USCustomary;
 
 @SuppressWarnings("all")
 public class MyRobot extends IterativeRobot {
+
+  private DistanceEncoder encoder;
+  private SpeedController motor;
+
+  private PIDController autoController;
+
   /**
    * This function is run when the robot is first started up and should be
    * used for any initialization code.
    */
   @Override
-  public void robotInit() {}
+  public void robotInit() {
+    encoder = new DistanceEncoder(0, 1, 2 * Math.PI, SI.METRE);
+    motor = new Spark(0);
+
+    autoController = new PIDController(0.5, 0, 0, encoder.value(), motor, SI.METRE);
```

Comment:
https://docs.google.com/document/d/12KhosAFriGCczBs6gwtJJDfg_QlANT92_lhxUWO2gCY/edit#heading=h.xw8d3vcow6xy

Reply:
While keeping up with standards is good, I think more people will expect meter cause that's how they spell it vs expecting metre because that's what a document says. Although, you could also do both and avoid any confusion.

Reply:
Although it does not make a difference, both already exist.

`Meter` is defined here: https://github.com/unitsofmeasurement/uom-systems/blob/master/common/src/main/java/systems/uom/common/USCustomary.java#L94